src/Hammer.js

```diff
@@ -126,6 +126,14 @@ var HammerComponent = React.createClass({
 			}
 		}, this);
 
+		var self = this;
+		props.ref = function(domElement) {
```

Comment:
@JedWatson @gwu 
cc @silentcloud 
This PR will cause  two problem:
1. break change, see below use case, the **children's string ref** will be override by Hammer
```js
        <Hammer
          direction="DIRECTION_HORIZONTAL"
          onPanStart={this.onPanStart}
          onPan={this.onPan}
          onPanEnd={this.onPanEnd}
        >
          <div className={`${prefixCls}-content`} ref="content">
            {children}
          </div>
        </Hammer>
```

2. cause react warning: https://facebook.github.io/react/warnings/special-props.html

Reply:
+1 Yes this is breaking `ref` for child components

Reply:
https://github.com/JedWatson/react-hammerjs/pull/87